scripts/run-module-bundlers-smoketests.js

```diff
@@ -8,7 +8,10 @@ Test webextension-polyfill bundled with webpack
 ===============================================`);
 
 process.env.TEST_BUNDLED_POLYFILL = "/tmp/webpack-bundle.js";
-result = shell.exec(`webpack --mode production --entry ./test/fixtures/bundle-entrypoint.js --output ${process.env.TEST_BUNDLED_POLYFILL}`).code ||
+const webpackOutputDir = process.env.TEST_BUNDLED_POLYFILL.replace(/\/[^/]+$/, "");
+const webpackOutputFilename = process.env.TEST_BUNDLED_POLYFILL.replace(/^.*\//, "");
```

Comment:
@Rob--W are these needed to get the dirname and basename for the webpack options? can't we use the methods from the nodejs `path` module instead?

Reply:
We could, yes.